lib/puppet/application/ssl.rb

```diff
@@ -74,6 +74,9 @@ def help
   `--localca` is specified, then also remove this host's local copy of the
   CA certificate(s) and CRL bundle. if `--target CERTNAME` is specified, then
   remove the files for the specified device on this host instead of this host.
+
+ * show
+  Print the full-text version of a host's certificate .
```

Comment:
nit: whitespace before period and colon missing after `* show`

also, maybe change to `Print the full-text version of this host's certificate.`